Source/Orts.Formats.Msts/CabViewFile.cs

```diff
@@ -598,6 +599,11 @@ public CVCDial(STFReader stf, string basepath)
                     ToDegree = stf.ReadFloat(STFReader.UNITS.None, null);
                     stf.SkipRestOfBlock();
                 }),
+                new STFReader.TokenProcessor("label", ()=>{
```

Comment:
The parameter should be called ortslabel since it does not exist in MSTS.

Reply:
There are recently added fields without the leading ORTS, like disablediflowvoltagepowersupplyoff and disabledifcabpowersupplyoff.

Reply:
Yeah... that might not have been a good idea.
In your case, you have other parameters around that are called ORTS<something> so it would make sense to use ORTS here.

Reply:
The reason why I didn't add "ORTS" is that this parameter was created as "Label" by Jindrich for ORNYMG, and there surely is content using it. If I switch to ORTSLabel I don't have a problem for ORNYMG, because I can make ORNYMG accepting both namings, but some existing content wouldn't show the label under the official OR versions.

Reply:
> The reason why I didn't add "ORTS" is that this parameter was created as "Label" by Jindrich for ORNYMG, and there surely is content using it. If I switch to ORTSLabel some existing content wouldn't show the label under the official OR versions.

So this is a conflict between 
- tidy code and documentation for official Open Rails
and
- content written and shared by early adopters using ORNYMG not showing correctly. 

I'm with Sharpe49 on this - Official Open Rails should aim for consistency, so content written for ORNYMG will have to change to work on Open Rails.

It will help, Carlo, if you go ahead and change ORNYMG to accommodate both forms as early adopters can run Open Rails-compatible content if they want to.

Reply:
OK, I've renamed "Label" to "ORTSLabel".

Reply:
Ok, thank you.